src/modules/mise.rs

```diff
@@ -0,0 +1,144 @@
+use super::{Context, Module, ModuleConfig};
+
+use crate::configs::mise::MiseConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module with the current mise config
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("mise");
+    let config = MiseConfig::try_load(module.config);
+
+    let mise_applies = !config.disabled
+        && context
+            .try_begin_scan()?
+            .set_extensions(&config.detect_extensions)
+            .set_files(&config.detect_files)
+            .set_folders(&config.detect_folders)
+            .is_match();
+
+    if !mise_applies {
+        return None;
+    }
+
+    let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        formatter
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(config.style)),
+                _ => None,
+            })
+            .map(|variable| match variable {
+                "symbol" => Some(Ok(config.symbol)),
+                "health" => match context.exec_cmd("mise", &["doctor"]) {
```

Comment:
One of the main things I would want to learn from this module is "do I need to run `mise install`" (ie are any tools missing), but it looks like that's not a "problem" that leads to `mise doctor` exiting non-zero.